packages/dds/tree/api-report/tree.alpha.api.md

```diff
@@ -104,6 +104,7 @@ export class FieldSchema<out Kind extends FieldKind = FieldKind, out Types exten
 export interface FieldSchemaMetadata<TCustomMetadata = unknown> {
     readonly custom?: TCustomMetadata;
     readonly description?: string | undefined;
+    readonly omitFromJson?: boolean | undefined;
```

Comment:
We shouldn't add this without a proper design review, especially since this is affecting a public API.

I am also personally not a fan of this pattern. I would prefer to extend the `toJsonSchema` transformation to allow the user to provide a callback that can be used to adjust the output based on their own custom metadata. This was the intention when `custom` was added here.

Reply:
Can our demo work okay without these changes? I'd prefer to have a separate PR add the relevant capabilities to the `toJsonSchema` transformation logic with the appropriate test coverage and everything. Would probably be easiest (if possible) to omit these changes for now and move forward with everything else.

Reply:
It can work. I've removed the omitFromJson changes in commit 7bfba91ce95a5532c21588b8e7e4b3b750edeecc

Reply:
Thanks! Let me know if/when you need the filtering functionality. I'm hoping that I'll be able to start drafting something on this next sprint, but I also thought I would get to it this sprint, so... 🤷‍♂️

Reply:
Tracking this work via this ticket: https://dev.azure.com/fluidframework/internal/_workitems/edit/22929/